core/UrlHelper.php

```diff
@@ -155,6 +155,14 @@ public static function getArrayFromQueryString($urlQuery)
         if (strlen($urlQuery) == 0) {
             return array();
         }
+
```

Comment:
Here sometimes 50% of the total time was spent while inserting requests in bulk!